### ISSUE TITLE
Rust: Version is not updated in dependant Cargo Workspace crate

### ISSUE THREAD

**Issue by andrey-yantsen** [google-github-actions/release-please-action#435]
### TL;DR

I have a project with four crates, two of which are handled by release-please-action. The second crate depends on the first, but its `Cargo.toml` does not get updated in release-please PR's.

### Action YAML

```yaml
name: release-please
jobs:
  release-please:
    runs-on: ubuntu-latest
    strategy:
      matrix:
        crate:
          - plex-api
          - plex-cli
    steps:
      - uses: google-github-actions/release-please-action@v3
        id: release
        with:
          path: "crates/${{ matrix.crate }}"
          package-name: ${{ matrix.crate }}
          release-type: rust
          monorepo-tags: true
          bump-minor-pre-major: true
          bump-patch-for-minor-pre-major: true
          changelog-types: |
            [{"type":"feat","section":"Features","hidden":false},
             {"type":"fix","section":"Bug Fixes","hidden":false},
             {"type":"chore","section":"Miscellaneous","hidden":false},
             {"type":"test","section":"Tests","hidden":true},
             {"type":"ci","section":"Continuous Integration","hidden":true},
             {"type":"docs","section":"Documentation","hidden":true},
             {"type":"refactor","section":"Code Refactoring","hidden":true},
             {"type":"style","section":"Styles","hidden":true}]
```

### Log output

```text
https://github.com/andrey-yantsen/plex-api.rs/runs/5167378607?check_suite_focus=true

https://pipelines.actions.githubusercontent.com/ELtVuWvm260Hc4dTvb6JM2rBUkUpxOGSn7BeoerUGNq8wS17uG/_apis/pipelines/1/runs/914/signedlogcontent/2?urlExpires=2022-02-12T12%3A44%3A28.4586251Z&urlSigningMethod=HMACV1&urlSignature=uxQGhY1p%2BOcmZ9mG7MsU7Uw8EQhTzujbzvI2oKUUQSc%3D
```

### Additional information

Here's the latest PR: andrey-yantsen/plex-api.rs#267. I expect the following line to be updated in the PR, but it doesn't: https://github.com/andrey-yantsen/plex-api.rs/blob/5ac0c2df33957c68d2d07487028c14cf6d2ce71e/crates/plex-cli/Cargo.toml#L13.

At the moment, I tend to think that I just missed some configuration steps. Still, it looks like release-please does not recognize the repo as a workspace — because `Cargo.lock` in the root of the repo should also be updated for the release, but again — it doesn't (here: https://github.com/andrey-yantsen/plex-api.rs/blob/5ac0c2df33957c68d2d07487028c14cf6d2ce71e/Cargo.lock#L1290).

**Comment by andrey-yantsen** [comment]
I'm guessing that I see this behaviour because of the set `path` parameter. I've configured the action as suggested here: https://github.com/google-github-actions/release-please-action/issues/180#issuecomment-756631682, but unfortunately, it doesn't work as expected for me.

**Comment by chingor13** [comment]
You are going to want to look into a "manifest" release configuration for handling multiple components. https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md
